client/src/index.js

```diff
@@ -1,23 +1,13 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import { BrowserRouter, Route, Switch } from "react-router-dom";
 import './index.css';
-import Homepage from "../src/Component/homepage/homepage";
-import SignUp from './Component/SignUp/SignUp';
-import SignIn from "./Component/SignIn/SignIn";
-import FeedPage from './components/feed';
-// import App from './App';
+import App from './App';
 import * as serviceWorker from './serviceWorker';
 
 ReactDOM.render(
-  <BrowserRouter>
-       <Switch>
-        <Route exact path="/" component={Homepage} />
-        <Route path="/signup" component={SignUp} />
-        <Route path="/feedpage" component={FeedPage} />
-        <Route path="/signin" component={SignIn} />
-      </Switch>
-  </BrowserRouter>,
+  <React.StrictMode>
```

Comment:
Change strictmode to BrowerRouter, otherwise "/" won't work. Add following code
```
import {BrowserRouter} from 'react-router-dom';

ReactDOM.render(
  <BrowserRouter>
    <App />
  </BrowserRouter>,
  document.getElementById('root')
);

```